tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -40,7 +40,7 @@ def clean_and_mkdir(dir_path):
 
 def _check_cpuid_x86(test_microvm, expected_cpu_count, expected_htt):
     expected_cpu_features = {
-        "maximum IDs for CPUs in pkg": f"{expected_cpu_count:#x} ({expected_cpu_count})",
+        "maximum IDs for CPUs in pkg": f"{32:#x} ({32})",
```

Comment:
I'm thinking that instead of just using the number 32, this should be extracted into a constant that can be used across tests, so that it can be easily updated should we support more vCPUs.